modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -503,6 +506,7 @@ object SeqexecEngine {
     val gcalKeywords            = cfg.require[Boolean]("seqexec-engine.gcalKeywords")
     val gnirsKeywords           = cfg.require[Boolean]("seqexec-engine.gnirsKeywords")
     val gpiKeywords             = cfg.require[Boolean]("seqexec-engine.gpiKeywords").fold(GPIKeywords.GPIKeywordsGDS, GPIKeywords.GPIKeywordsSimulated)
+    val gpiGDS                  = Uri.unsafeFromString(cfg.require[String]("seqexec-engine.gpiGDS"))
```

Comment:
I'm surprised there's not already a `URI` decoder that lets us do `.require[URI]` ... should we add one?

Reply:
Could be, in the same line as for `Site`